### ISSUE TITLE
[Feature suggestion] Global volume slider to pause menu?

### ISSUE THREAD

**Issue by borbware** [nesbox/TIC-80#1613]
It would be nice to have a global volume slider in the pause menu.

**Comment by nesbox** [comment]
Seems the pause menu has to be redesigned, a lot of requests about the pause menu recently :)

**Comment by nesbox** [comment]
The menu items will look like this:

```
RESUME GAME
RESET GAME
CLOSE GAME

OPTIONS >

    FULLSCREEN:     ON/OFF
    CRT MONITOR:    ON/OFF
    VOLUME:         0-100%

    GAMEPAD > 
        [1][2][3][4]
        UP
        DOWN
        LEFT
        RIGHT
        A
        B
        X
        Y

    KEYBOARD >
        LAYOUT: [AZERTY QWERTY QWERTZ QZERTY ...]
        [][][][][][][][][][][]
        [][][][][][][][][][][]
        [][][][][][][][][][  ]
        [][][][        ][][][]

QUIT TIC-80
```

**Comment by nesbox** [comment]
It will be fixed here #1613

**Comment by nesbox** [comment]
`NO_SOUND` option replaced with the `VOLUME` option in the config and cmd params 072c742
